### PR TITLE
Posts and like api integration

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me! You can merge the PR

File: src/components/PostDropdown/PostDropdown.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  position: fixed;
+  width: fit-content;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
```
Comment (Developers-codz) [inline src/components/PostDropdown/PostDropdown.css:12]: Using Css color variable will be good ?

File: src/pages/Auth/Login/Login.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  color: var(--text-primary);
+}
+
+.login_checkbox_inp_container {
+  align-items: center;
+  gap: 0.4rem;
+}
+
+.login_inp_label_resize {
+  font-size: 0.95rem;
```
Comment (Developers-codz) [inline src/pages/Auth/Login/Login.css:30]: for such a small value u can use px

**Review by Developers-codz — APPROVED** [review]
Both UI and code looks good to me, You can go for merge
Just few suggestions below

File: src/pages/Auth/Login/Login.css (hunk 2 above)
Comment (abhisheksahani-nova) [reply src/pages/Auth/Login/Login.css:30]: yahh sure

File: src/components/PostDropdown/PostDropdown.css (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/components/PostDropdown/PostDropdown.css:12]: Yess I will do that

**Pull request merged**